kedro-databricks-bootcamp/03_intermediate/rocketfuel/notebooks/Understanding_Hooks.ipynb

```diff
@@ -23,6 +23,33 @@
     "Hooks Concepts: https://docs.kedro.org/en/stable/hooks/introduction.html"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "87e0cbdd",
+   "metadata": {},
+   "source": [
+    "Kedro uses `pluggy`, a popular hooks framework for Python originally created for `pytest`. Pluggy hooks work as follows:\n",
```

Comment:
Took the liberty of adding some extra context here, @merelcht feel free to delete or ignore

Reply:
@ravi-kumar-pilla wrote that part, but lgtm 👍